src/resources/beta/chat/completions.ts

```diff
@@ -119,19 +120,19 @@ export class Completions extends APIResource {
   runTools<
     Params extends ChatCompletionToolRunnerParams<any>,
     ParsedT = ExtractParsedContentFromParams<Params>,
-  >(body: Params, options?: Core.RequestOptions): ChatCompletionRunner<ParsedT>;
+  >(body: Params, options?: RunnerOptions): ChatCompletionRunner<ParsedT>;
```

Comment:
This type has been wrong such that it causes a TS error when you try to pass `maxCompletions`. Will Stainless auto-gen remove my change? If so, what's the right way to fix this? It's been driving me nuts 😅

Reply:
Ah thanks for fixing! No the generator will leave this change in :)

Would you mind opening a separate PR with this change so we can get it merged faster?

Reply:
Good call! Will update now...

Reply:
https://github.com/openai/openai-node/pull/1073